addons/mail/static/src/models/messaging/messaging.js

```diff
@@ -8,6 +8,12 @@ function factory(dependencies) {
 
     class Messaging extends dependencies['mail.model'] {
 
+        _created() {
+            this.initializedPromise = new Promise(resolve => {
+                this._initializedPromiseResolve = resolve;
+            });
```

Comment:
Could you define them as proper fields?
The more we use model/field API, the better.

Reply:
For the promise, ok to add as field. For the resolve I'll just remove it and make a deferred with resolve directly on itself.